works.css

```diff
@@ -161,6 +163,75 @@ h1 {
   margin-left: 5px;
 }
 
-ul {
-  list-style-type: none;
+.language-name {
+  padding-top: 20px;
+}
+
+.bt {
+  display: flex;
+}
+
+#contact-me {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  border-top-left-radius: 50px;
+  padding: 1rem;
+  background-image: url("./assets/contact\ form\ shapes-mobile.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: initial;
+}
+
+#contact-head {
+  margin-top: 8rem;
+  font-weight: bold;
+  font-size: 40px;
+}
+
+#contact-description {
+  margin: 1rem;
+  color: white;
+  font-size: 18px;
+  font-style: normal;
+  display: flex;
+  text-align: center;
+}
+
+#form {
+  margin: 1rem;
+}
+
+#name {
+  width: 100%;
+  height: 43px;
+  margin-bottom: 1rem;
+  border-radius: 9px;
+  border: 1px solid rgb(206, 218, 216);
+}
+
+#email {
+  width: 100%;
+  height: 43px;
+  margin-bottom: 1rem;
+  border-radius: 9px;
+  border: 1px solid rgb(206, 218, 216);
+}
+
+#message {
+  width: 100%;
+  margin-bottom: 1rem;
+  border-radius: 9px;
+  border: 1px solid rgb(206, 218, 216);
+}
```

Comment:
- [ ] Always use the class attribute for multiple elements (do not use ID selector for multiple elements).
- [ ] Here you are not following the DRY principle. name, email, and message has same property, you can avoid repeating this using class and use that class in HTML wherever you need.

Reply:
Resolved this issue!